core/src/main/java/org/apache/calcite/sql/validate/ListScope.java

```diff
@@ -221,9 +232,15 @@ public RelDataType resolveColumn(String columnName, SqlNode ctx) {
     case 1:
       return type;
     default:
+      // Only return type if unknown tables are allowed, and at most one of the matched tables
```

Comment:
nit: Line wrap at 80 characters for this comment line and the rest in this block.

Reply:
Oops, accidentally set wrapping to 100.